alluxio/alluxio_file_system.py

```diff
@@ -137,6 +139,21 @@ def __init__(
             worker_addresses, self.logger
         )
         self.http_port = http_port
+        self.req_counter = 0
+        self.threadLock = threading.Lock()
+        def refresh_worker():
+            with self.threadLock:
+                if self.req_counter == 0:
```

Comment:
if there are always requests coming in, then we will never refresh worker list?
and also using a request counter may not always be correct. Some users may call partial of a method and then cancel